resources/lrsql/config/config.edn

```diff
@@ -37,6 +37,17 @@
    :prod-sqlite     #include "prod/default/connection.edn"
    :prod-sqlite-mem #include "prod/default/connection.edn"
    :prod-postgres   #include "prod/postgres/connection.edn"}
+ :tuning
+ #profile
```

Comment:
We discussed this before: while I, as a developer, think that any tuning config should be subordinate to overall DB config, having tuning config be top-level may be easier to grasp for end users. Is that the reasoning behind putting tuning config at the top?

Reply:
So I'm trying to look at the category more broadly. There may in fact be CODE changes in the future that adjust the performance profile of an LRS for different use cases.

Reply:
Ah, so code changes that have nothing to do with adjusting the DDL or the queries?

Reply:
I could see it. Batching strategies, timeout stuff, etc. oooo i just realized that txn-retries might be a good candidate to be moved here. What do you think?

Reply:
Also it should be noted that milt is building some pretty heavy duty stuff that's gonna get tacked on and may itself have some tuning properties.